test/controllers/ReviewPenaltyController.test.ts

```diff
@@ -141,16 +164,20 @@ describe('ReviewPenaltyController', () => {
             penaltyIdentifier: {
                 companyNumber,
                 companyName,
-                userInputPenaltyReference: penaltyReference,
+                penaltyReference,
                 penaltyList: { items: [{ id: 'A0000001' }, { id: 'A0000002' }] }
             } as PenaltyIdentifier
         };
 
-        const app = createApp({
-            appeal: appeal as Appeal, navigation: {
-                permissions: [REVIEW_PENALTY_PAGE_URI]
-            }
-        });
+        const app = createApp(
```

Comment:
Not a required change but we could pull out this constant and reuse it in the other methods. There are 4 methods that resolve the hasExistingAppeal to false and only one to true

Reply:
I see what you mean, I personally like having that sort of information present in each test, makes it easy to understand how its all set up for that particular test, what do you think @n-r-b